src/pages/patterns/index.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: Design overview
+description: >
+  Foundations for creating purposeful UI.
```

Comment:
@allisonacs, @ctancinco I think this will need some work.

Reply:
Let me know what's working/not working and I can give a few options.

Reply:
@ctancinco this was copied from the existing [Design overview](https://garden.zendesk.com/design) page. This new "Patterns overview" will need its own tagline description that aligns with the other top-level pages.

Reply:
Got it, thank you. Pattern overview options:
- Commonly used components and how they’re used within the product
- Commonly used components for better UX
- Product problems and the components that love them
- Existing design solutions for a more consistent UX

Reply:
- "Commonly" feels ... common.
- "problems" feels ... problematic.
- "solutions" feels right.

Solutions feels more aligned with content:principles, design:foundations, component:instructions.

What do you think, @allisonacs? Or should we riff offline?

Reply:
+1, I like solutions. Let's go with that.

Reply:
Probably helpful to note that every other hi-level IA page description is reflected on the home page. I would expect this to be the same.

Reply:
I've updated this to `Existing design solutions for a more consistent UX`. Thanks for the help.